Makefile

```diff
@@ -19,7 +19,7 @@ build-dir := build
 
```

Comment:
Could you please update the python-style submodule to bring also the config change for pytest in?

Reply:
Sure, done.